registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/config/WorkflowManagerConfigBeans.java

```diff
@@ -80,6 +82,11 @@ public WebSubUtil getWebSubUtil() {
 	public RestTemplate getRestTemplate() {
 		return new RestTemplate();
 	}
+	
+	@Bean
+	public AnonymousProfileService getAnonymousProfileService() {

```

Comment:
u dont need this. I think jyoti alrady added in RegistrationStatusBeanConfig.java

Reply:
we will need this because if we run workflow service independently that time it will throw an error if not added...

Reply:
workflow-manager-service has dependency of registration-processor-registration-status-service-impl.
The config should be scanned and AnonymousProfileService should get instantiated. 
You can remove from here and test once.

Reply:
yes.. removed and it is working pls check again...